src/test/data/JsonSerializableMeetingsBookTest/typicalMeetingsMeetingsBook.json

```diff
@@ -0,0 +1,21 @@
+{
+  "meetings" : [ {
+    "title" : "CS2103 Meeting",
+    "link" : "https://zoom.sg",
+    "startTime" : "2022-10-10 1800",
```

Comment:
test with real links as well

> https://nus-sg.zoom.us/j/92307270969?pwd=VVMvNWFPWFpyVHRIcXR0VkJlNkg0dz09

Reply:
I think real links are tested in `LinkTest`?

Reply:
oh i didnt know about that class, but i think it will be good to test storing and retrieval of actual links

Reply:
but this is the typical meetings book. If we change the link here, the whole test needs to change

Reply:
I realized that there's no need. I can use the real link